docs/admin/hardware.rst

```diff
@@ -22,18 +22,65 @@ In order to print submissions, a supported non-networked printer is required. Su
 
 More printer options will be added in future releases.
 
+Lenovo T series Laptops
+-----------------------
+
+Lenovo ThinkPad T490 (with 8th generation Intel Core processor)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The Thinkpad T490 **with an 8th generation Intel Core processor** is a recommended option for the SecureDrop Workstation. If you plan to use it, you should follow the instructions below to ensure that the BIOS is up to date and adequately configured before proceeding with the installation.
+
+
+.. caution::
+
+  There versions of the T490 with 10th generation Intel Core processors, and are at present **untested and unsupported**. The workstation has been tested on model 20N2002AUS.
+
+Network devices (Ethernet and Wi-Fi) will not immediately work out of the box and require a one-time manual configuration on install. After Qubes starts for the first time, ``sys-net`` will fail to start:
+
+|screenshot_sys_net_pci_reset|
+
+Open a ``dom0`` terminal via **Q > Terminal Emulator**, and run the following command to list the devices connected to the ``sys-net`` VM.
+
+.. code-block:: sh
+
+  qvm-pci ls sys-net
+
+
+This will return the two devices (Ethernet and WiFi) that are connected to ``sys-net``:
+
+.. code-block:: sh
+
+  BACKEND:DEVID  DESCRIPTION                                                            USED BY
+  dom0:00_14.3   Network controller: Intel Corporation                                  sys-net
+  dom0:00_1f.6   Ethernet controller: Intel Corporation Ethernet Connection (5) I219-V  sys-net
+
+
+For both device IDs (e.g. ``dom0:00_1f.6`` and ``dom0:00_14.3``), you will need to detach and re-attach the device to ``sys-net``, then restart ``sys-net`` as follows:
+
+.. code-block:: sh
+
+  qvm-pci detach sys-net dom0:00_14.3
+  qvm-pci detach sys-net dom0:00_1f.6
+  qvm-pci attach sys-net --persistent --option-no-strict-reset=True dom0:00_14.3
+  qvm-pci attach sys-net --persistent --option-no-strict-reset=True dom0:00_1f.6
```

Comment:
Based on testing just now, should be `s/option-/option=/`

Reply:
good catch, fixed. Based on the message board post in the commit message i believe it should be `--option no-strict-reset=True`, fixed !